.github/workflows/integration-linux.yml

```diff
@@ -151,7 +151,7 @@ jobs:
             action-validator 0.1.2
       - name: Lint Actions
         run: |
-          find .github/workflows -type f \( -iname \*.yaml -o -iname \*.yml \)             | xargs -I {} action-validator --verbose {}
+          find .github/workflows -maxdepth 1 -type f \( -iname \*.yaml -o -iname \*.yml \)             | xargs -I {} action-validator --verbose {}
```

Comment:
💡 Question\Observation Please: I noticed we added `maxdepth` to `1` depth here will mean go a level deep but in current `dir structure` there is not level deep as all the `yaml` files reside under same level. Why is this `maxdepth 1` needed please? Thanks

Current dir structure of target is this:

![790609ED-9571-47A7-ACFB-BE05F5500E37](https://user-images.githubusercontent.com/6233295/229333697-ca23952f-efa8-4e16-b0f1-d678658ce8ff.jpeg)

Reply:
I think this runs with the intention to find the workflow within `./langtest/*` as that is the destination provided within the generate workflow command. I believe that it will work here as intended but also dont believe its necessary. If we create more tests later on in a deeper destination directory we would need to update this command as well.